frontend/packages/console-dynamic-plugin-sdk/src/extensions/topology-details.ts

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { Extension } from '@console/plugin-sdk/src/typings/base';
+import { GraphElement } from '@patternfly/react-topology';
+import { CodeRef, EncodedCodeRef, UpdateExtensionProperties } from '../types';
+
+namespace ExtensionProperties {
```

Comment:
Ideally these definitions are contained in the package that owns them. But for now putting them in the main SDK is fine.